release-notes/release-notes54.md

```diff
@@ -23,7 +23,22 @@ These release notes provide an overview of the changes for {{site.productname}}
 
 The following new features and enhancements were added for the {{site.productname}} 5.4 release.
 
-### 
+### Updates and additions to the Table plugin
+
+The {{site.productname}} 5.4 release includes the following improvements for the table (`table`) plugin:
+
+- Adds: commands, APIs, toolbar buttons, icons, and menu items for; cut, copy, and paste columns.
+- Adds toolbar buttons and icons for cut, copy, and paste rows.
```

Comment:
The toolbar buttons for rows technically already existed, so all we've done is add icons.